src/components/InformationPanes.js

```diff
@@ -107,19 +116,41 @@ const RightSidebar = React.createClass({
       });
     }
 
-    return items;
+    return items.filter(item => item);
+  },
+
+  renderHorizontalLayout() {
+    return Accordion({
+      items: this.getItems()
```

Comment:
this might be a stretch, but I could see separate pane functions: 

```js
renderHorizontalLayout() {
  return Accordion({
    items: [
      this.scopesPane(),
      this.framesPane(),
    ]
  })
}
```